docker/start-op-challenger.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -e
+
+# how do we want to handle network-dependant configs here?
+DISPUTE_GAME_FACTORY=0x5a20cD16d4E51D3B2A6b2Ab460Ac735Fe421b820
+L2_OUTPUT_ORACLE=0xE6Dfba0953616Bacab0c9A8ecb3a9BBa77FC15c0
+
+echo "Starting op-challenger"
+
+exec op-challenger \
+    --l1-ws-endpoint ${L1_WS_RPC_URL} \
+    --trusted-op-node-endpoint http://magi:${RPC_PORT} \
+    --signer-key $CHALLENGER_SIGNER_KEY \
```

Comment:
Alright, I've made this optional in the challenger and updated the setup to include the mode, so that we can run it in `listen-and-respond` mode easily in the future

I've also added an option to turn this service off entirely from the Magi setup. I figured maybe it's something that not everybody could want. Let me know if you'd like to have it turned on or off as default!